algorithms/kernel/low_order_moments/oneapi/low_order_moments_batch_oneapi_impl.i

```diff
@@ -191,13 +189,17 @@ LowOrderMomentsBatchTaskOneAPI<algorithmFPType, scope>::LowOrderMomentsBatchTask
 
     nColsBlocks = (nFeatures + maxWorkItemsPerGroup - 1) / maxWorkItemsPerGroup;
 
-    nRowsBlocks = 8;
+    nRowsBlocks = 128;
```

Comment:
Is it possible to put in a separate function depending on the dimensions and returned nBlock?

Reply:
Yes, it will be rewritten during transition to new interfaces.